### PR TITLE
Updating to angular2-*@2.0.0-rc6 (deps, specs ...)

### PR DESCRIPTION
**Pull request by katallaxie**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Angular 2 RC6

* **What is the current behavior?** (You can also link to an open issue here)

Angular 2 RC5

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by katallaxie** [comment]
@gdi2290 I have updated to RC6. There seems to be an issue with the `DedupePlugin`. Should we move ahead anyway?

**Comment by gdi2290** [comment]
@katallaxie there always seems to be a problem with `DedupePlugin`. I would leave a comment saying it doesn't work atm otherwise developers would be confused if they uncomment it

File: src/app/+detail/index.ts [hunk 1]
```diff
@@ -17,7 +17,6 @@ export const routes = [
     Detail
   ],
   imports: [
-    BrowserModule,
     FormsModule,
```
Comment (gdi2290) [inline src/app/+detail/index.ts]: don't you need `CommonModule` here?

Comment (katallaxie) [inline src/app/+detail/index.ts]: I have included it again. But `BrowserModule` is already imported in `app.module.ts`.

**Pull request merged**